.all-contributorsrc

```diff
@@ -138,6 +138,15 @@
       "contributions": [
         "doc"
       ]
+    },
+    {
+      "login": "jsjoeio",
+      "name": "JavaScript Joe",
+      "avatar_url": "https://avatars3.githubusercontent.com/u/3806031?v=4",
+      "profile": "https://jsjoe.io",
+      "contributions": [
+        "talk"
```

Comment:
happy if you're happy, but does `tutorial` fix an egghead course better that `talk`? 
 You can see all the options [here](https://allcontributors.org/docs/en/emoji-key).

Reply:
Ah, I didn't see `tutorial` - that makes more sense. Good catch! Let me update that